common/app/views/fragments/containers/facia_cards/container.scala.html

```diff
@@ -28,6 +28,17 @@
              }
              data-component="@componentName"
              aria-expanded="true">
+
+             @containerDefinition.container match {
+                 case Dynamic(DynamicPackage) => {
+                     <style>
```

Comment:
Was thinking about this last night, do you think we should attempt to [scope](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/style#attr-scoped) these styles?